Dockerfile

```diff
@@ -36,6 +36,20 @@ RUN mix phx.digest && mix release
 
 FROM debian:buster-slim AS app
 
+ARG CLIENT_ID=:sfractal2020
```

Comment:
can we use :sbompoc for this repo (ie don't use same for each repo)?

Reply:
Yeah, sure
```suggestion
ARG CLIENT_ID=:sbompoc
```